misc/scripts/upgrade.sh

```diff
@@ -142,6 +142,7 @@ else
 	done < /tmp/pacstall-up-urls
 
 	export local='no'
+	cd $SRCDIR
```

Comment:
```suggestion
	cd "$SRCDIR"
```

Fixes codacy issue

Reply:
Also shouldn't we `cd` back to the user's original directory after this operation is complete?

Reply:
cd'ing in a script only works inside the script. What happens inside the script, stays in the script